### ISSUE TITLE
why not making __file_to_ast method as a class ????

### ISSUE THREAD

**Issue by fanifieiev** [yegor256/aibolit#131]
I see a lot of methods which look identical in the project
```
def __file_to_ast(self, filename: str) -> javalang.ast.Node:
        """
        Takes path to java class file and returns AST Tree
        """
        with open(filename, encoding='utf-8') as file:
            tree = javalang.parse.parse(file.read())

        return tree
```
Why not creating a separate class:
```
class Ast:
       def __init__(self, filename: str):
           self.filename = filename
   
       def value() -> javalang.ast.Node:
            with open(filename, encoding='utf-8') as file:
               tree = javalang.parse.parse(file.read())
            return tree
```

**Comment by 0crat** [comment]
@acheshkov[/z](https://www.0crat.com/u/acheshkov) please, pay attention to this issue

**Comment by acheshkov** [comment]
@0crat in

**Comment by 0crat** [comment]
The job [#131](https://github.com/yegor256/aibolit/issues/131) assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/CT5UB7XAP/5a120df8-b846-45e3-bd7d-19e7b4c8ebdb); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by 0pdd** [comment]
@fanifieiev 3 puzzles [#158](https://github.com/yegor256/aibolit/issues/158), [#159](https://github.com/yegor256/aibolit/issues/159), [#160](https://github.com/yegor256/aibolit/issues/160) are still not solved.

**Comment by fanifieiev** [comment]
The issue has been resolved, closing the ticket

**Comment by 0crat** [comment]
Job was finished in 27 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0pdd** [comment]
@fanifieiev all 3 puzzles are solved here: [#158](https://github.com/yegor256/aibolit/issues/158), [#159](https://github.com/yegor256/aibolit/issues/159), [#160](https://github.com/yegor256/aibolit/issues/160).
